SignalMessaging/categories/UIView+OWS.m

```diff
@@ -111,6 +111,14 @@ - (NSLayoutConstraint *)autoVCenterInSuperview
     return [self autoAlignAxis:ALAxisHorizontal toSameAxisOfView:self.superview];
 }
 
+- (void)autoPinEdgesToEdgesOfView:(UIView *)view
```

Comment:
In line with pure layout conventions, these auto* methods should return the constraints they create.

Also, how is this different from `autoPinToEdgesOfView` which already does that?

Reply:
Removed.